src/Filesystem/F.php

```diff
@@ -598,15 +598,11 @@ public static function niceSize(
 	 */
 	public static function read(string $file): string|false
 	{
-		if (
-			is_readable($file) !== true &&
-			Str::startsWith($file, 'https://') !== true &&
-			Str::startsWith($file, 'http://') !== true
-		) {
+		try {
+			return @file_get_contents($file);
+		} catch (Throwable) {
 			return false;
 		}
```

Comment:
`Issue`: I don't think we can use `try catch` here. This only works in the CMS context because Whoops converts PHP warnings (such as "No such file or directory") into exceptions. But as soon as Whoops is disabled, there won't be an exception, so the `catch` won't execute and the method will return `void`, which will then be a fatal error because of the type hinting and might also break expectations in other parts of the system.

Instead of `try catch` with error suppression, it would be a lot cleaner to use `Helpers::handleErrors()`. The question is: Does that have a significant performance impact? It at least does not do I/O, so I guess it won't, but we should verify that.

Reply:
I've tried wrapping it in handleErrors, but the warnings are still created without the error controller.

Reply:
Could you push your try? If we suppress the "No such file or directory" specifically in the handler callback but let all other warnings and errors be thrown, I think that would be exactly what we need.

Reply:
The thing here really is that we cannot surpress the warnings in any other way as far as I can see. The controller is the best bet and it still keeps the results in line with what we had so far. We return false if the file cannot be read and the content if it can.

Reply:
Pushed a working version, see Discord